cmd/gotk/get_helmrelease.go

```diff
@@ -68,7 +69,7 @@ func getHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 		}
 		isInitialized := false
 		for _, condition := range helmRelease.Status.Conditions {
-			if condition.Type == helmv2.ReadyCondition {
+			if condition.Type == meta.ReadyCondition {
```

Comment:
This could also use `meta.GetCondition()` but looking at the logic in general, it could use an overall update to be more accurate.